osu.Game/Rulesets/UI/ClickableModIcon.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Please no `nullable disable` on new classes

Reply:
These two class is copied from `ModIcon` and `ModDisplay`

I think we can use my processing method in these two classes

But not within the scope of pr, I didn't change them